.github/workflows/run-tests.yml

```diff
@@ -48,10 +47,50 @@ jobs:
           go-version: "1.20"
           check-latest: true
 
-      - name: Build dependencies
+      - name: Install Go dependencies
         run: |
-          make deps:modules
-          make deps:test
-      
+          go mod download
+          go install gotest.tools/gotestsum@latest
+          go install github.com/ory/go-acc@latest
+
+      - name: Build Lens dependencies
```

Comment:
suggestion: However, I think it should probably be done via a `make` command, instead of defined directly in the github action file

Reply:
Fixed